### PR TITLE
Support aiidalab/full-stack

### PR DESCRIPTION
**Pull request by csadorf**
- [ ] ~Implement migration path~
- [x] Implement tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,46 @@ Please see [here](ssh-forward.md) for instructions on how to run AiiDAlab on a r
 
 This package follows the Python compatibility and deprecation schedule specified by [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html).
 
+## Development
+
+### Set up a development environment
+
+We recommend to use conda/mamba to setup a new Python environemnt and then install the development and test dependencies with
```
Comment (danielhollas) [inline README.md:77]:
```suggestion
We recommend to use conda/mamba to setup a new Python environment and then install the development and test dependencies with
```

Comment (csadorf) [reply README.md]: Thanks for catching that! 👍

**Comment by csadorf** [comment]
This PR appears ready to me, however some of the tests seemingly randomly time out. I'd like to at least try to figure out what the root cause might be before introducing this kind of flakiness into the main branch.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -26,12 +26,15 @@ jobs:
...
-        timeout-minutes: 10
+        timeout-minutes: 30
 
         strategy:
             fail-fast: false
             matrix:
                 python-version: ['3.8', '3.9', '3.10']
+                default-image:
+                    - ''  # use the application default
+                    - aiidalab/aiidalab-docker-stack:latest
```
Comment (unkcpz) [inline .github/workflows/ci.yml:37]: But `aiidalab/aiidalab-docker-stack` image is built from the old way and only supports 1.6? Am I missing something?

**Review by unkcpz — COMMENTED** [review]
@csadorf Thanks a lot for working on this! I think all great, but I didn't run it to test. But I think we can get it merged and I will do the actual run on my computer. 
Only some minor questions may be better to clarified before merging.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (csadorf) [reply .github/workflows/ci.yml:37]: A newer version of aiidalab-launch can still run the old image. That is what we are testing here.

File: aiidalab_launch/instance.py [hunk 3]
```diff
@@ -270,7 +271,7 @@ def exec_create(self, cmd: str, privileged: bool = False) -> str:
             return self.client.api.exec_create(
                 self.container.id,
                 cmd,
-                user=None if privileged else self.profile.system_user,
+                user="root" if privileged else self.profile.system_user,
```
[the remark this answers is not in the document]
Comment (csadorf) [reply aiidalab_launch/instance.py:274]: I am sorry, but I don't understand your comment. Can you try to clarify it, please?

[the remark this answers is not in the document]
Comment (unkcpz) [reply aiidalab_launch/instance.py:274]: Don't feel sorry, it's my fault. I already Imaged that you will ask me to clarify it when I writing it😬 . I am on the way to office, we can discuss in person.

**Review by unkcpz — APPROVED** [review]

File: aiidalab_launch/profile.py [hunk 4]
```diff
@@ -30,7 +31,7 @@ def _default_port() -> int:  # explicit function required to enable test patchin
     return DEFAULT_PORT
 
 
-_DEFAULT_IMAGE = "aiidalab/aiidalab-docker-stack:latest"
+_DEFAULT_IMAGE = "aiidalab/full-stack:edge"
```
Comment (csadorf) [inline aiidalab_launch/profile.py:34]:
```suggestion
_DEFAULT_IMAGE = "aiidalab/full-stack:latest"
```

**Pull request merged**